### DISCUSSION TITLE
Add bindings to Info-mode-map too

### DISCUSSION THREAD

**Discussion by jdtsmith** [kickingvegas/casual-info#8] [Ideas]
Thanks.  I really like the opinionated bindings here; I'm always confused by n/p.  It would be useful to have an option or function to update the bindings in `Info-mode-map` itself too (possibly by default, on load).  It's odd to have n/p do something different whether the transient is popped-up or not.

**Comment by kickingvegas** [comment]
@jdtsmith Thanks for sharing your thoughts! I’ll consider making a function, but have you seen the customization section in the [README.org](https://github.com/kickingvegas/casual-info/blob/main/README.org) file? There it shows how to bind `n` and `p` to the functions used in Casual Info.

**Reply by jdtsmith** [reply]
I did see that after I posted and implemented it, thanks.  A function might be a good idea so that way if you later pick a slightly different binding there would be consistency.
